test/integration/kubernetes_deploy_test.rb

```diff
@@ -930,12 +941,18 @@ def test_resource_watcher_reports_failed_after_timeout
     end
     assert_deploy_failure(result)
 
+    bad_probe_timeout = if KUBE_SERVER_VERSION < Gem::Version.new("1.10.0")
+      "Deployment/bad-probe: TIMED OUT (timeout: 5s)"
+    else
+      "Deployment/bad-probe: GLOBAL WATCH TIMEOUT (20 seconds)"
```

Comment:
In this version, `bad-probe` has no purpose since `missing-volumes` is already covering the global watch timeout.